invenio_cli/commands/packages.py

```diff
@@ -37,6 +37,45 @@ def install_packages(packages):
 
         return steps
 
+    @staticmethod
+    def outdated_packages():
+        """Steps to show outdated packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        prefix = ['pipenv', 'update']
+        cmd = prefix + ['--outdated']
+
+        steps = [
+            CommandStep(
+                cmd=cmd,
+                env={'PIPENV_VERBOSITY': "-1"},
+                message="Checking outdated packages..."
+            )
+        ]
+
+        return steps
+
+    @staticmethod
+    def update_packages(packages):
+        """Steps to update all or some Python packages.
+
+        It is a class method since it does not require any configuration.
+        """
+        cmd = ['pipenv', 'update']
+        for package in packages:
+            cmd.extend([package])
```

Comment:
hmm why do you need to loop and extend one by one? can't it be done directly as `cmd.extend(packages)`?

In any case, after the changes requested above you will need to check if for packages:

```python
if packages:
    cmd.extend(packages)
```